x/programs/runtime/dependencies.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runtime
+
+import (
+	"context"
+)
+
+type Runtime interface {
```

Comment:
Probably should also expose something like `GetBalance(context.Context) uint64`.

Reply:
will add as a followup